nsq/writer.py

```diff
@@ -131,7 +135,7 @@ def _pub(self, command, topic, msg, callback):
         conn.callback_queue.append(callback)
         cmd = getattr(protocol, command)
         try:
-            conn.send(cmd(topic, msg))
+            conn.send(cmd(topic, msg) if delay_ms is None else cmd(topic, delay_ms, msg))
```

Comment:
this is unfortunate, we could probably clean it up, but it doesn't matter that much I guess